thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/template.hbs

```diff
@@ -43,7 +43,7 @@
     applyAction=(action "onRange")
   }} --}}
   <!-- Investigation range slider -->
-  <div class="date-slider">
+  <div class="date-slider date-slider--offset">
```

Comment:
Looks good. Is there a reason not to fix this on the `.date-slider ` class itself vs creating a new modifier class?

Reply:
not really, just figured the slider may be re-used somewhere else